src/screens/Home/index.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+
+import BookList from 'screens/BookList';
 
 import Navbar from '../../components/Navbar';
 
 function Home() {
-  return <Navbar />;
+  return (
+    <Fragment>
+      <Navbar />
+      <BookList />
+    </Fragment>
+  );
```

Comment:
You can do this, to avoid import Fragment
```
<>
    <Navbar />
    <BookList />
 </>
```
https://reactjs.org/docs/fragments.html#short-syntax